src/views/users/User/UserList.vue

```diff
@@ -65,7 +66,16 @@ export default {
             }
           },
           mfa_level: {
-            width: '130px'
+            width: '130px',
+            formatter: (row) => {
+              if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 1) {
+                return this.$t('MFAAllUsers')
+              } else if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 2 && (row?.is_superuser || row?.is_org_admin)) {
+                return this.$t('MFAOnlyAdminUsers')
+              } else {
+                return row['mfa_level'].label
+              }
+            }
           },
           source: {
             width: '120px',
```

Comment:
The code has been updated to include the necessary imports. The changes suggest no major discrepancies, but it could be beneficial to clean up unnecessary lines of code that do not add value.

No immediate issues were found in terms of syntax or logic integrity; however, optimizing the component may involve adjusting data fetching strategies to improve performance, potentially reducing load times especially when handling extensive records like invite users dialog components which rely heavily on API calls for user fetch.